_config.yml

```diff
@@ -1,4 +1,4 @@
 name: Your New Jekyll Site
 markdown: redcarpet
-pygments: true
-baseurl: /figure
+highlighter: null
+baseurl: /figure
```

Comment:
OK, think I get it. Lets leave it for now (been like that since forever anyway).